plugins/modules/resource_governor.py

```diff
@@ -7,21 +7,21 @@
 DOCUMENTATION = r'''
 ---
 module: resource_governor
-short_description: Configures the resource governor on a SQL Server instance.
+short_description: Configures the resource governor on a SQL Server instance
 description:
-     - Enables or disables and optionally sets the classifier function for the resource governor feature.
+- Enables or disables and optionally sets the classifier function for the resource governor feature.
 version_added: 0.1.0
 options:
   enabled:
     description:
-      - Whether to enable or disable resource governor.
+    - Whether to enable or disable resource governor.
     type: bool
     required: false
     default: true
   classifier_function:
     description:
-      - The name of the classifier function that resource governor will use. To clear the function, use the string 'NULL'.
-      - Sending a literal $null will not change the value.
+    - The name of the classifier function that resource governor will use. To clear the function, use the string C(NULL).
+    - Sending a literal C($null) will not change the value.
```

Comment:
This seems like it applies only to the underlying PowerShell function, should it be removed?
```suggestion
```